io.openems.shared.influxdb/src/io/openems/shared/influxdb/InfluxConnector.java

```diff
@@ -168,6 +151,24 @@ public InfluxConnection(InfluxDBClient client, WriteApiBlocking writeApi) {
 		}
 	}
 
+	/**
+	 * Returns a debug log of the current influx connection state.
+	 * 
+	 * @return the debug log string
+	 */
+	public String debugLog() {
+		return new StringBuilder("[monitor] ") //
```

Comment:
why only `"[monitor] "` instead of `"[InfluxDB] [monitor] "` ?

Reply:
now its `[Timedata.InfluxDB] timedata0 [monitor]  ....`
https://github.com/OpenEMS/openems/pull/2440/files/287290e4c45c77bcebe6a366b8066ceb92513b0a#diff-59be03f4f524ebe54c9905e895f97b2365cdb165d28dad80f474ed2f7cd51349R443

because i removed the component id from InfluxConnector